### PR TITLE
Add 1.23.0-RC1 release notes

### PR DESCRIPTION
**Pull request by upadhyeammit**
Added release notes for RC1.

### CODE REVIEW

**Comment by upadhyeammit** [comment]
@tbrisker what should be in headline features, I just tried to get all features fixed in 1.23 and sort it out as per priority to see if we have anything which was high priority to add it in headline features but all seems low and normal, so maybe we have different criteria to consider feature as headline, more details on same would be helpful. If you have any more suggestion please comment, will push changes accordingly. Thank You!

**Comment by ares** [comment]
I forgot, can we also list plugin changes here? I would have a significant change related to ansible using ansible-runner by default.

File: _includes/manuals/1.23/1.23.0-RC1_release_notes.md [hunk 1]
```diff
@@ -0,0 +1,303 @@
+## Release notes for {{page.version}}
+
+## Headline features
+
+### Release Notes for 1.23.0-RC1
```
Comment (tbrisker) [inline _includes/manuals/1.23/1.23.0-RC1_release_notes.md]:
```suggestion
### Release Notes for 1.23.0
```

File: _includes/manuals/1.23/1.23.0-RC1_release_notes.md [hunk 2]
```diff
@@ -0,0 +1,303 @@
...
+* Update new guest OS type supported on VMware vSphere 6.7 and above ([#26838](https://projects.theforeman.org/issues/26838))
+* VMware compute resource cannot find VM folders when datacenters are themselves nested in folders ([#25849](https://projects.theforeman.org/issues/25849))
+* Associating VMs/hosts with multiple VMWare compute resources from same vcenter server ([#15626](https://projects.theforeman.org/issues/15626))
+
+#### Compute resources - oVirt
+* Remove hardcoded max values of oVirt CR ([#27334](https://projects.theforeman.org/issues/27334))
+* Remove "experimental" from ovirt v4 checkbox ([#26840](https://projects.theforeman.org/issues/26840))
+* Allow Satellite user to choose between VNC, SPICE for Display type on RHEV compute resources ([#24532](https://projects.theforeman.org/issues/24532))
+
+#### Core
```
Comment (tbrisker) [inline _includes/manuals/1.23/1.23.0-RC1_release_notes.md]: These should be moved to `#### Smart proxies` section, as they are the core component of smart proxy

File: _includes/manuals/1.23/1.23.0-RC1_release_notes.md [hunk 3]
```diff
@@ -0,0 +1,303 @@
...
+* unneeded asset compile for hidden_values.js and lookup_keys.js ([#26647](https://projects.theforeman.org/issues/26647))
+* Improve Layout active MenuItems performance ([#26581](https://projects.theforeman.org/issues/26581))
+* add react-router foundation ([#26186](https://projects.theforeman.org/issues/26186))
+* Vertical navigation missing IDs ([#26052](https://projects.theforeman.org/issues/26052))
+* Add components from katellos 'move_to_foreman' directory to foreman ([#25888](https://projects.theforeman.org/issues/25888))
+* Refactor the FactChart component structure to group files by domain ([#25864](https://projects.theforeman.org/issues/25864))
+* Move LoginPage into a React component using patternfly-react LoginPage component. ([#24990](https://projects.theforeman.org/issues/24990))
+* Move gridster from bundle to npm ([#16287](https://projects.theforeman.org/issues/16287))
+
+#### foreman-installer script
```
Comment (tbrisker) [inline _includes/manuals/1.23/1.23.0-RC1_release_notes.md]: These should move to the `#### Installer` section

**Review by tbrisker — CHANGES_REQUESTED** [review]
See a few comments inline. Also, these should be added in https://github.com/theforeman/theforeman.org/blob/gh-pages/_includes/manuals/1.23/1.2_release_notes.md and not as a new file.

**Comment by upadhyeammit** [comment]
> I forgot, can we also list plugin changes here? I would have a significant change related to ansible using ansible-runner by default.

I think we can, @tbrisker can you please confirm once. Either give me details about changes and I shall add it, or maybe you can create new pr once this gets merged.

File: _includes/manuals/1.23/1.23.0-RC1_release_notes.md (hunk 3 above)
Comment (upadhyeammit) [reply _includes/manuals/1.23/1.23.0-RC1_release_notes.md]: done!

File: _includes/manuals/1.23/1.23.0-RC1_release_notes.md (hunk 2 above)
Comment (upadhyeammit) [reply _includes/manuals/1.23/1.23.0-RC1_release_notes.md]: done!

**Review by tbrisker — APPROVED** [review]
Thanks @upadhyeammit !

**Comment by tbrisker** [comment]
@ares we don't normally list plugin changes in the foreman manual. Perhaps a blog post and/or the plugin's release notes would be a more appropriate way to publicize the change?

**Pull request merged**